packaging/imagemode/Makefile

```diff
@@ -0,0 +1,139 @@
+#
+# The following variables can be overriden from the command line
+# using NAME=value make arguments
+#
+PULL_SECRET ?= $(HOME)/.pull-secret.json
+BASE_IMAGE_URL ?= registry.redhat.io/rhel9/rhel-bootc
+BASE_IMAGE_TAG ?= 9.4
+
+#
+# Define the main targets
+#
+all:
```

Comment:
Should `all`, `rhocp` and other target have `.PHONY`? Outcomes of these targets are not files named like these targets, so I guess they should?